web/styles/left_sidebar.css

```diff
@@ -313,6 +313,73 @@
     }
 }
 
+.stream_search_section {
+    position: sticky;
+    top: 0;
+    /* Must be more than .stream-list-subsection-header */
+    z-index: 3;
```

Comment:
This is a follow-up for after this PR gets merged, but the use of `z-index` in situations like this is regrettable. What we need, I think, is to better organize elements in the DOM and use `position: sticky` with thoughtful `top:` values that allow things to stack up the way we want them to.

Reply:
Yeah this is mostly piggybacking on what was already there, but agreed that it would be better to not use `z-index`.